apps/deploy-web/src/pages/deployments/[dseq]/index.tsx

```diff
@@ -1,19 +1,25 @@
+import { z } from "zod";
+
 import { DeploymentDetail } from "@src/components/deployments/DeploymentDetail";
+import { CI_CD_TEMPLATE_ID } from "@src/config/remote-deploy.config";
+import { getValidatedServerSideProps } from "@src/lib/nextjs/getValidatedServerSIdeProps";
+import { services } from "@src/services/http/http-server.service";
 
-type Props = {
-  dseq: string;
-};
+export default DeploymentDetail;
 
-const DeploymentDetailPage: React.FunctionComponent<Props> = ({ dseq }) => {
-  return <DeploymentDetail dseq={dseq} />;
-};
+const contextSchema = z.object({
+  params: z.object({
+    dseq: z.string().regex(/^\d+$/)
+  })
+});
 
-export default DeploymentDetailPage;
+export const getServerSideProps = getValidatedServerSideProps(contextSchema, async ({ params }) => {
+  const remoteDeployTemplate = await services.template.findById(CI_CD_TEMPLATE_ID);
```

Comment:
looking into the logic in the components it seems like this template is still needed there. Perhaps this could be changed ofc.

But I was thinking of how often that template is changed? perhaps let's move it to static props and call it a day?